plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/messages.properties

```diff
@@ -74,6 +74,7 @@ projectselector.project.not.selected=Select a project
 projectselector.filter=Filter projects by name or ID
 projectselector.retrieveproject.error.message=An error occurred while retrieving projects: {0}
 projectselector.retrieveapplication.error.message=An error occurred while retrieving App Engine application in project {0}:\n{1}
+projectselector.retrieveapplication.permissionerror.message=This account does not have permission for the App Engine application in project {0}.
```

Comment:
I was testing this out, and I wonder if it might be better to put the account name here too?

Reply:
_'briandealwis@gmail.com' does not have permission to access the App Engine application in project 'cloud-tools-for-eclipse-kokoro'_ seems better to me.

Reply:
That would be nice. I don't see the account available in this method though, and I didn't want to make big changes just to push it in here. Is there some easy way to grab it from a credential object?

Reply:
We don't seem to be able to get that easily :-(